tests/ballerina-integration-test/src/test/java/org/ballerinalang/test/util/TestUtils.java

```diff
@@ -164,4 +171,17 @@ public static void prepareBalo(Object test) throws URISyntaxException {
         Path target = Paths.get(path).getParent();
         System.setProperty(BLangConstants.BALLERINA_HOME, target.toString());
     }
+
+    public static KeyStore getKeyStore(File keyStore)
+            throws IOException, KeyStoreException, CertificateException, NoSuchAlgorithmException {
+        KeyStore ks = null;
+        String tlsStoreType = "PKCS12";
+        if (keyStore != null) {
+            try (InputStream is = new FileInputStream(keyStore)) {
+                ks = KeyStore.getInstance(tlsStoreType);
```

Comment:
```suggestion
                ks = KeyStore.getInstance("PKCS12");
```

And remove the variable declaration?